erpnext/hr/doctype/employee_compensation/employee_compensation.js

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2021, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Employee Compensation', {
+	// refresh: function(frm) {
+
+	// }
+});
```

Comment:
@sahilk25 you need to move the code as well if present, after changing the module of the doctype!
```suggestion
frappe.ui.form.on('Employee Compensation', {
	validate: function(frm) {
		if (frm.doc.date_of_injury > frappe.datetime.get_today()) {
			frappe.throw(__("You cannot select a future date for Date of Injury"));
		}
		if (frm.doc.first_aid_date > frappe.datetime.get_today()) {
			frappe.throw(__("You cannot select a future date for First Aid Date"));
		}
		if (frm.doc.first_aid_date < frm.doc.date_of_injury) {
			frappe.throw(__("The First Aid date cannot be before the Date of Injury"));
		}
	}
});
```